litedict.py

```diff
@@ -70,22 +73,30 @@ def __init__(
         self.conn.execute(f"PRAGMA cache_size = {-1 * 64_000};")
 
     def __setitem__(self, key, value):
-
+        if self.writeback:
+            self._writeback[key] = value
         self.conn.execute(
             "INSERT OR REPLACE INTO  Dict VALUES (?, ?)", (key, self.encoder(value))
         )
 
     def __getitem__(self, key):
-        c = self.conn.execute("SELECT value FROM Dict WHERE Key=?", (key,))
-        row = c.fetchone()
-        if row is None:
-            raise KeyError(key)
-        return self.decoder(row[0])
+        try:
+            value = self._writeback[key]
+        except KeyError:
+            c = self.conn.execute("SELECT value FROM Dict WHERE Key=?", (key,))
+            row = c.fetchone()
+            if row is None:
+                raise KeyError(key)
+            value = self.decoder(row[0])
+            if self.writeback:
+                self._writeback[key] = value
+        return value
 
     def __delitem__(self, key):
-
+        self._writeback.pop(key,None) # Will not fail even if missing
```

Comment:
Line not formatted using `black`.

Reply:
I didn't run anything through black. I didn't realize it was being used. I'd be surprised if this was the only example! I'll `black` it.